### ISSUE TITLE
Positive infinity with an explicit plus is not parsed correctly

### ISSUE THREAD

**Issue by pavel-kirienko** [dpranke/pyjson5#63]
```python
>>> import json5  # v0.9.11

>>> json5.loads('Infinity')  # Correct.
inf

>>> json5.loads('-Infinity')  # Correct.
-inf

>>> json5.loads('+123.456')  # Correct.
123.456

>>> json5.loads('+Infinity')  # Non-spec behavior, incorrect; expected inf.
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "~/.local/lib/python3.10/site-packages/json5/lib.py", line 81, in loads
    raise ValueError(err)
ValueError: <string>:1 Unexpected "I" at column 2
<string>:1 Unexpected "I" at column 2
```

**Comment by dpranke** [comment]
Hi! Thanks for reporting this and I apologize for the delay in getting back to you. I will try get to this soon.

**Comment by dpranke** [comment]
Fixed in ff2a072201fd4ec3cf1783e1ab0ecec6d3e7cb94.

Thanks again for filing the bug and sorry again for me not getting to it before now.
